crates/ark/src/modules/positron/html_widgets.R

```diff
@@ -9,28 +9,49 @@
     # Render the widget to a tag list.
     rendered <- htmltools::as.tags(x, standalone = TRUE)
 
-    # Resolve the dependencies for the widget (scripts, stylesheets, etc.).
-    dependencies <- htmltools::resolveDependencies(
-        attr(rendered, "html_dependencies", exact = TRUE))
+    # Render the tag list to a temporary file using html_print. Don't view the
+    # file yet; we'll do that in a bit.
+    tmp_file <- htmltools::html_print(rendered, viewer = NULL)
+
+    # Derive the height of the viewer pane from the sizing policy of the widget.
+    is_plot <- x$sizingPolicy$knitr$figure
+    height <- x$sizingPolicy$viewer$paneHeight
+    if (identical(height, 'maximize')) {
+        height <- -1
+    } else if (is.null(height)) {
+        height <- 0
+    }
```

Comment:
For both `is_plot` and `height`, maybe add additional checks that they are bool-like and integer-like respectively? If they aren't, may result in a rust panic. Could have default values if we detect weirdness

Reply:
Sure, done!